frontend/src/LineGraph.res

```diff
@@ -181,6 +181,13 @@ let make = React.memo((
 ) => {
   let graphDivRef = React.useRef(Js.Nullable.null)
   let graphRef = React.useRef(None)
+
+  // Dygraph does not display the last tick, so a dummy value
+  // is added a the end of the data to overcome this.
+  // See: https://github.com/danvk/dygraphs/issues/506
+  let lastData = data[Belt.Array.length(data) - 1]
```

Comment:
This is a nit: `lastRow` or `lastDatapoint` would probably be a little bit more clear here?

Reply:
Also what if `data` is empty?